packages/store/addon/-private/ts-interfaces/minimum-serializer-interface.ts

```diff
@@ -0,0 +1,280 @@
+/**
+  ## Overview
+
+  In order to properly manage and present your data, `EmberData`
+  needs to understand the structure of data it receives.
+
+  `Serializers` convert data between the format used by an API
+  and the format `EmberData` understands.
+
+  Data received from an API response is `"normalized"` into 
+  [JSON:API](https://jsonapi.org/) (the format used internally
+  by `EmberData`), while data sent to an API is `"serialized"`
+  into the format the API expects.
+
+  ### Implementing a Serializer
+
+  There are only two required serializer methods, one for
+  normalizing data in `JSON:API`, and another for serializing
+  records via `Snapshot`s into the expected API.
+
+  To implement a serializer, export a class implementing the
+  [MinimumSerializerInterface](MinimumSerializerInterface) from
+  the `app/serializers/` directory. An example is below.
+
+  ```ts
+  import EmberObject from '@ember/object';
+
+  export default class ApplicationSerializer extends EmberObject {
+    normalizeResponse(_, __, rawPayload) {
+      return rawPayload;
+    }
+    serialize(snapshot, options) {
+      const serializedResource = {
+        id: snapshot.id(),
+        type: snapshot.modelName,
+        attributes: snapshot.attributes()
+      };
+
+      return serializedResource;
+    }
+  }
+ ```
+
+
+  #### Serializer Resolution
```

Comment:
Can you add an example to this section as well? I think it would really help things gel.  

Thinking something like:

```
store.serializerFor('author');

// lookup paths (in order) =>

app/serializers/author.js
app/serializers/application.js
```